tools/cli/commands/deleteStack.js

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs';
+import chalk from 'chalk';
+import { deleteStack } from '../helpers/util';
+
+import { STACK_MAP, BASE_PATH } from '../config';
+
+/**
+ * Delete a list of technologies
+ *
+ * @param {Array} stackList - The list of technologies
+ * @param {Function} logger - The Logger
+ * @param {Boolean} isShowStackList - The flag to show list of existing technologies
+ */
+const handleDeleteStackList = (stackList, logger, isShowStackList) => {
+  // getting a list of existing technologies
+  const existsStackList = fs
+    .readdirSync(`${BASE_PATH}/packages`)
+    .filter(stack => Object.keys(STACK_MAP).includes(stack))
+    .map(stack => STACK_MAP[stack].name);
+
+  // check for the availability of a flag if there is,
+  // show a list of technologies
+  if (isShowStackList) {
```

Comment:
`delete` and `list` operations shouldn't be combined. It's confusing.

Reply:
done